D/DDSCAT/build_tarballs.jl

```diff
@@ -0,0 +1,37 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "DDSCAT"
+version = v"7.3.3"
+
+# Collection of sources required to complete build
+sources = [
+    ArchiveSource("http://ddscat.wikidot.com/local--files/downloads/ddscat$(version)_220120.tgz", "06f2673a45fcff20b8ed9f37d5bedeb84604d7a9fb10f627096833163769d1e7")
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd $WORKSPACE/srcdir/src
+make ddscat
+mv ddscat "ddscat${exeext}" || true
+install -Dvm 755 "ddscat${exeext}" "${bindir}/ddscat${exeext}"
+install_license ../doc/UserGuide.pdf
```

Comment:
The only information about the license in this PDF file is at page 7 (of 108...):

> You comply with the “copyleft" agreement (more formally, the GNU General Public License) of
> the Free Software Foundation: you may copy, distribute, and/or modify the software identified as
> coming under this agreement. If you distribute copies of this software, you must give the recipients
> all the rights which you have. See the file `doc/copyleft.txt` distributed with the DDSCAT
> software.

which isn't incredibly informative because (1) there are multiple versions of the GPL license, and (2) there is no `doc/copyleft.txt` file whatsoever.

I don't think adding a PDF file as license file which only casually and inaccurately mentions the license is very useful.  We have the actual text of a couple of GPL licenses:

```suggestion
install_license /usr/share/licenses/GPL-2.0+
```
or
```suggestion
install_license /usr/share/licenses/GPL-3.0+
```
Which is the right one is everyone's guess.

Reply:
I am not sure but I think we can use GPL-3.0+ since I see a [Python wrapper of DDSCAT](https://pythonhosted.org/ScatPy/#license) used it.